frontend/src/utils/prettyNumbers.test.ts

```diff
@@ -24,3 +24,22 @@ describe("prettyNumbers", () => {
     });
   });
 })
+
+describe("amountToString", () => {
+  test("pn()", () => {
+    [
+      {input: null, output: undefined},
```

Comment:
@KoalaSat just a heads up. After merging this PR the frontend build GH action was failing because it did not pass this test. I amended this test in the main branch.